src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/readcount/SimpleCountCollection.java

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.readcount;
+
+import org.broadinstitute.hdf5.HDF5File;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SampleLocatableCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SampleMetadata;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SimpleSampleMetadata;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+
+import java.io.File;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Simple data structure to pass and read/write a List of {@link SimpleCount} objects.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public final class SimpleCountCollection extends SampleLocatableCollection<SimpleCount> {
+    enum SimpleCountTableColumn {
```

Comment:
You can make this private

Reply:
These have been package-protected in the other SampleLocatableCollection classes, so I'll just keep it as is to be consistent.  At some point we may actually want to make all of these public, instead.